RS-Project10/Screen.java

```diff
@@ -0,0 +1,248 @@
+        System.out.println("Enter length of triangle");
+        length = sc.nextInt();
+        listOfParameters.add(0, length);
+        if (!operation.checkCoordinates(xCoordinate + length, yCoordinate + length) && 
+            !operation.checkCoordinates(xCoordinate, yCoordinate)) {
+            System.out.println("coordinates are wrong");
+            System.exit(0);
+        }
+        if (!operation.checkCoordinates(pointXCoordinate, pointYCoordinate)) {
+            System.out.println("coordinates are wrong");
+            System.exit(0);
+        }
+        listOfShapes.add(ShapesFactory.createTriangle("Triangle", origin, pointOppositeToBase, 
+            listOfParameters));
+    }
+
+    /**
+     * create circle objects
+     * @param list of shapes
+     */
+    public void addCircle(ArrayList<Shapes> listOfShapes) {
+        int radius;
+        Point origin;
+        int xCoordinate, yCoordinate;
+        Scanner sc = new Scanner(System.in);
+        ShapesOperations operation = new ShapesOperations();
+        ArrayList<Integer> listOfParameters = new ArrayList<Integer>();
+        System.out.println("Enter center x coordinate");
+        xCoordinate = sc.nextInt();
+        System.out.println("Enter center y coordinate");
+        yCoordinate = sc.nextInt();
+        origin = new Point(xCoordinate, yCoordinate);
+        System.out.println("Enter radius of circle");
+        radius = sc.nextInt();
+        if (!operation.checkCoordinates(xCoordinate + radius, yCoordinate + radius) && 
+            !operation.checkCoordinates(xCoordinate - radius, yCoordinate - radius)) {
+            System.out.println("coordinates are wrong");
+            System.exit(0);
+        }
+        listOfParameters.add(0, radius);
+        listOfShapes.add(ShapesFactory.createShape("Circle", origin, listOfParameters));
+    }
+
+    public static void main(String[] args) {
+        int choice;
+        int option;
+        Scanner string = new Scanner(System.in);
+        Screen screen = new Screen();
+        ArrayList<Shapes> listOfShapes = new ArrayList<Shapes>();
+        ShapesOperations operation = new ShapesOperations();
+        do {
+            System.out.println("Graphics library");
+            System.out.println("Choose option");
+            System.out.println("1. To add shapes to screen");
+            System.out.println("2. To delete shape from screen");
+            System.out.println("3. To delete particular shape type from screen");
+            System.out.println("4. To display list of shape on screen");
+            System.out.println("5. To sort shapes in ascending order");
+            System.out.println("6. To get shapes enclosing specific point");
+            System.out.println("7. To get all shapes above given shape");
+            System.out.println("8. To get origin of shape");
+            Scanner sc = new Scanner(System.in);
+            option = sc.nextInt();

```

Comment:
if user entering the wrong input, your code should handle that with a message. Like currently if I am entering a string instead of Int the program is failing.